Libraries/TestHelpers.psm1

```diff
@@ -372,7 +372,7 @@ Function Run-LinuxCmd([string] $username, [string] $password, [string] $ip, [str
 		}
 	}
 	if ($global:sshPrivateKey) {
-		Write-LogDbg ".\Tools\plink.exe -ssh -t -i ppkfile -P $port $username@$ip $logCommand"
+		Write-LogDbg ".\Tools\plink.exe -ssh -t -i $sshKey -P $port $username@$ip $logCommand"
```

Comment:
I update it into a fix and fake value since $sshKey is a long string - the full path of private key file.

Reply:
I added it for easier debugging, reverted back now.